app/src/test/java/com/fcaputo/parkingapp/modelTests/SettingsModelTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.fcaputo.parkingapp.settingsTests
+package com.fcaputo.parkingapp.modelTests
```

Comment:
a package should never be in camel case. It should be just model as you are in the tests

Reply:
Done